src/components/TipInput.vue

```diff
@@ -121,6 +123,25 @@ export default {
         };
       },
     }),
+    largestFtTipAmount() {
+      return this.tipUrlStats.tokenTotalAmount.length
+        ? [...this.tipUrlStats.tokenTotalAmount].sort((a, b) => (
```

Comment:
Actually, you don't need to sort array to get the max value, I would use `reduce` instead 🤪
```suggestion
        ? this.tipUrlStats.tokenTotalAmount.reduce(
          (a, b) => a.amount > b.amount ? a : b, // use the right comparison from the below
          this.tipUrlStats.tokenTotalAmount[0],
        )
```